src/asset_manager/config/config.js

```diff
@@ -63,6 +63,9 @@ module.exports = {
   // }
   uploadFile: '',
 
+  // In the absence of uploadFile function assets will be embedded as Base64
+  embedAsBase64: 1,
```

Comment:
Somehow I've missed to add it to the config file, thanks.
BTW we need it false as default. For a correct setup you just have to provide the upload endpoint but with this option as true the uploading will not work (it will be even a breaking change), so we'll get a lot of complaints that uploading is not working